test.sql

```diff
@@ -0,0 +1,8 @@
+\c nc_games_test
+SELECT reviews.*, 
+    CAST(COUNT(*)AS int) AS comment_count 
+    FROM reviews 
```

Comment:
same as above this does not need to be commit up to github

Reply:
Think I've deleted them now